Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -673,7 +677,24 @@ extension ResourceFileElements {
                     }
                 )
             }
-        )
+        }
+
+        func handleCopyRule() -> [ProjectDescription.ResourceFileElement] {
+            let copyResources = resources.filter { $0.rule == .copy }
+
+            let resourcesPaths = copyResources.map { path.appending(RelativePath($0.path)) }
+
+            return resourcesPaths.map { absolutePath in
+                return .folderReference(path: Path(absolutePath.pathString))
+            }
+        }
+
+        let resourceFileElements = handleProcessRule() +  handleCopyRule()
```

Comment:
I would rather have a single loop and switch over `$0.rule`, so we are sure we handle all scenarios

Reply:
Included a switch statement to ensure all scenarios are handled